libsel4vmmplatsupport/src/plat/zynqmp/devices/vuart.c

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
Can this device name be updated to indicate which device model it is emulating?

Reply:
sure thing!